src/components/Breadcrumb/index.js

```diff
@@ -3,11 +3,24 @@ import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
 import { connect } from 'react-redux'
 import { msg } from '_/intl'
+import { HomeIcon } from '@patternfly/react-icons'
 
 const buildPath = (vms, branches) => {
-  const res = []
+  const res = [{
+    title: (<HomeIcon style={{ fontSize: '1rem' }} />),
+    url: '/',
+  }]
 
   for (const branch of branches) {
+    const { match: { path = '' } = {} } = branch
```

Comment:
The purpose of `routes.js` is to have a readable overview of  all routes. Dynamic path replacement with hardcoded paths goes in  the opposite direction.

I would expect following structure in `routes.js`
1. root (home) with path/URL `/`. It has 2 sub-branches:VM Machines and Account Settings. Clicking on the root should redirect to VM Machines.
2.  VM Machines with path `/vms` (note the change `vm` -> `vms`).  Single VM would use path `/vms/{id}`.
3. Account Settings with path `/settings` (no change here)

Reply:
@sjd78 
Please share you opinion. Should we model this fix via branch structure(`routes.js`) or is the fix in the visual layer enough?

Reply:
@bamsalem I agree with Radek that we should put all the branches logic in routes.js.
Is it a complicated fix to do?

Reply:
Not at all, you can review the changes :)